Source/VCExpr/VCExprASTVisitors.cs

```diff
@@ -1234,7 +1234,7 @@ public virtual VCExpr Visit(VCExprVar node, Arg arg)
         }
       }
 
-      if (!changed)
+      if (!changed) 
```

Comment:
Please eliminate the change to this file.

Reply:
Done. Thanks a lot for your comments!